src/implementations/vanilla/components/expandable/_expandable.scss

```diff
@@ -16,11 +16,30 @@ $_expandable-content-color: null !default;
   transform: rotate(0deg);
 }
 
+.ecl-expandable__divider {
+  background-color: map.get(theme.$color, 'grey-25');
+  border: none;
+  display: none;
+  height: 1px;
+  margin: 0;
+}
+
 .ecl-expandable__content {
   color: $_expandable-content-color;
-  font-family: #{map.get(theme.$font-family, 'default')};
+  font: map.get(theme.$font-prolonged, 'm');
+  margin-top: map.get(theme.$spacing, 'm');
```

Comment:
To limit the risks of double margin, we could remove the margin top of the first element. That's what has been done on ECL4
https://github.com/ec-europa/europa-component-library/pull/3008/files#diff-a5765bf8fb7136b374a2c627867a3c0463f88b0b4cb4504a4b405c1c35cbd92aR33

Reply:
i did it, but it's only going to work as long as the margins are not set through an utility class. I considered the case of multiple <p> so resetting margin-top of the first and margin-bottom of the last, if there is only one both margins will be reset